src/github.com/getlantern/flashlight/config/fetcher.go

```diff
@@ -131,7 +130,7 @@ func (cf *fetcher) fetchCloudConfig(cfg *Config) ([]byte, error) {
 	// successive requests
 	req.Close = true
 
-	resp, err := cf.httpFetcher.Do(req)
+	resp, err := cf.rt.RoundTrip(req)
```

Comment:
It just depends on what features we need.  http.Client adds redirect handling, cooky handling and probably some other stuff, but I don't think that's needed in this particular spot.